jupyterhub_traefik_proxy/kv_proxy.py

```diff
@@ -299,7 +299,8 @@ async def delete_route(self, routespec):
         status, response = await self._kv_atomic_delete_route_parts(
             jupyterhub_routespec, route_keys
         )
-
+        if response is None:
+            return
```

Comment:
If  `_kv_atomic_delete_route_parts` returns a `True` status, then we don't need to exit early as it wouldn't fail anymore.
```suggestion
```